mbus-api/src/common/retry.rs

```diff
@@ -0,0 +1,91 @@
+use std::{cmp::min, time::Duration};
+
+/// Backoff delay options for the retries to the NATS jetstream message bus
+/// Max number of retries until it gives up.
+#[derive(Clone, Debug)]
+pub struct BackoffOptions {
+    // initial delay
+    pub init_delay: Duration,
```

Comment:
since the variable are public, the comments must have 3 backslashes, so that it converts to doc comments.